docs/blockchain_data/confirmed_transactions.md

```diff
@@ -4,24 +4,24 @@ Up until this point, we've explored how to notify LDK of chain activity using
 blocks. But what if you're sourcing chain activity from a place that doesn't
 provide a block-centric interface, like Electrum?
 
-LDK's `ChannelManager` and `ChainMonitor` have a `chain::Confirm` interface to
-support this use case, analogous to the block-oriented `chain::Listen`
-interface which we've been using up until now. With this alternative approach,
-you still need to give LDK information about chain activity, but only for
-transactions of interest. These are identified by
-`Confirm::get_relevant_txids`, as well as by `Filter`'s
-`register_tx`/`register_output` methods, just as before.
+LDK's `ChannelManager` and `ChainMonitor` implement a `chain::Confirm` interface to support this
+use case, analogous to the block-oriented `chain::Listen` interface which we've been using up until
+now. With this alternative approach, you still need to give LDK information about chain activity,
+but only for transactions of interest. To this end, LDK needs to be notified when any transactions
```

Comment:
Avoid the passive voice here and tighten up as:

To this end, you must call `Confirm::transactions_confirmed` when any transactions identified by `chain::Filter`'s `register_tx`/`register_output` methods are confirmed.

Reply:
Done with e6dac60.